src/styles/global.scss

```diff
@@ -111,5 +126,16 @@ body {
 
   .demo-section {
     padding: $spacing-small;
+
+    .button {
+      height: 50px;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      * {
+        font-size: 24px;
```

Comment:
Do we need this if we use the text components that have sizing attributes associated with them?

Reply:
Ahh I see. It looks like the button component we have take a string. Maybe we should update it so that it can wrap a `<Text />` component?